src/main/resources/com/google/api/codegen/py/sample.snip

```diff
@@ -107,7 +107,7 @@
                 @if class.pageStreaming.isResourceMap
                     for {@class.pageStreaming.resourceKeyVarName}, {@class.pageStreaming.resourceValueVarName} in {@class.responseVarName}['{@class.pageStreaming.resourceFieldName}'].items():
                         @# TODO: Change code below to process each ({@class.pageStreaming.resourceKeyVarName}, {@class.pageStreaming.resourceValueVarName}) item:
-                        pprint({@class.pageStreaming.resourceKeyVarName}, {@class.pageStreaming.resourceValueVarName})
+                        pprint({{@class.pageStreaming.resourceKeyVarName}: {@class.pageStreaming.resourceValueVarName}})
```

Comment:
I'd prefer my [original formulation](https://github.com/saicheems/toolkit/blob/8d9d0bd9e3059f8b20ffef6051635c7d7a85c8a1/src/main/resources/com/google/api/codegen/py/discovery_fragment.snip) that uses a tuple instead --- it seems too misleading to print a series of single-item dictionaries to represent a single map.